modules/mobile/app/controllers/mobile/v0/claims_and_appeals_controller.rb

```diff
@@ -20,12 +22,24 @@ class ClaimsAndAppealsController < ApplicationController
         end
       end
 
-      before_action(except: %i[index get_claim request_decision]) do
+      before_action(except: %i[index get_claim request_decision upload_document upload_multi_image_document]) do
```

Comment:
At this point, perhaps it's worth changing this to an `only` because i think it only applies to one method now.

Reply:
That does seem to be the case...